index.js

```diff
@@ -0,0 +1,192 @@
+// Task 1
+/* Функция принимает массив пользователей. Исходные данные см. ниже. Возвращает объект, состоящий из двух полей: female и male. Данные поля являются массивами, в которых содержаться пользователи, подходящие по категории gender соответственно. Также, вместо двух полей first_name и last_name у каждого из объектов должно быть поле fullName в котором будут объеденины убранные поля (first_name и last_name). Количество пользователей может быть не ограничено */
+
+const users = [
+  {
+    id: 1,
+    first_name: "Jeanette",
+    last_name: "Penddreth",
+    email: "jpenddreth0@census.gov",
+    gender: "female",
+    ip_address: "26.58.193.2",
+  },
+  {
+    id: 2,
+    first_name: "Petr",
+    last_name: "Jackson",
+    email: "gfrediani1@senate.gov",
+    gender: "male",
+    ip_address: "229.179.4.212",
+  },
+];
+
+const UserGenderArray = (users) => {
```

Comment:
issue: in according with JS code style, you have to name your functions in `camelCase` not in `StartCase`.
`StartCase` is applicable only for `classes`

fix the naming of your functions here and in the code below

Reply:
fixed